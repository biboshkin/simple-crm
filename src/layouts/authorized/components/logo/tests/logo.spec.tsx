import React from "react";
import renderer from "react-test-renderer";

import { Logo } from "..";

test("render correct with collapsed false", () => {
  const component = renderer.create(<Logo collapsed={false} />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});

test("render correct with collapsed true", () => {
  const component = renderer.create(<Logo collapsed={true} />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
