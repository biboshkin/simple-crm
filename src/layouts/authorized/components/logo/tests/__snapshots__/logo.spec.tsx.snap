// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct with collapsed false 1`] = `
<div
  className="logo"
>
  <img
    alt="logo"
    className="img"
    src="logo.png"
  />
  <span
    className="ant-typography title"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    <strong>
      БИЗНЕС РЕСУРС
    </strong>
  </span>
</div>
`;

exports[`render correct with collapsed true 1`] = `
<div
  className="logo"
>
  <img
    alt="logo"
    className="img"
    src="logo.png"
  />
  <span
    className="ant-typography title collapsed"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    <strong>
      БИЗНЕС РЕСУРС
    </strong>
  </span>
</div>
`;
